assets/compatcheck.js

```diff
@@ -0,0 +1,15 @@
+const addonBtn = document.querySelector("a.home__extension-link");
```

Comment:
I guess this should be wrapped in something like `window.addEventListener("DOMContentLoaded"` to avoid touching the DOM before it's settled